stripe/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -202,14 +208,16 @@ class Stripe internal constructor(
         confirmPaymentIntentParams: ConfirmPaymentIntentParams,
         stripeAccountId: String? = this.stripeAccountId
     ) {
-        paymentController.startConfirmAndAuth(
-            AuthActivityStarter.Host.create(fragment),
-            confirmPaymentIntentParams,
-            ApiRequest.Options(
-                apiKey = publishableKey,
-                stripeAccount = stripeAccountId
+        fragment.lifecycleScope.launch {
```

Comment:
can limit it further to its viewLifecycleOwner
`fragment.viewLifecycleOwner.lifecycleScope.launch`

Reply:
I think `lifecycleScope` is probably fine as long as I'm not accessing the Fragment's view?